openapi-contracts/src/test/java/com/sinch/sdk/domains/verification/models/dto/v1/webhooks/VerificationResponseEventDtoTest.java

```diff
@@ -20,31 +20,39 @@
 @TestWithResources
 public class VerificationResponseEventDtoTest extends BaseTest {
 
-  public static CalloutRequestEventResponseDto expectedCalloutRequestEventResponseDto =
-      new CalloutRequestEventResponseDto()
-          .action(VerificationEventResponseActionDto.DENY)
-          .callout(
-              new CalloutRequestEventResponseCalloutDto()
-                  .code("4567")
-                  .speech(new CalloutRequestEventResponseCalloutSpeechDto().locale("the locale")));
-  public static FlashCallRequestEventResponseDto expectedFlashCallRequestEventResponseDto =
-      new FlashCallRequestEventResponseDto()
-          .action(VerificationEventResponseActionDto.DENY)
-          .flashCall(
-              new FlashCallRequestEventResponseFlashCallDto().cli("cli code").dialTimeout(123));
-  public static SMSRequestEventResponseDto expectedSMSRequestEventResponseDto =
-      new SMSRequestEventResponseDto()
-          .action(VerificationEventResponseActionDto.DENY)
-          .sms(
-              new SMSRequestEventResponseSmsDto()
-                  .code("5666")
-                  .acceptLanguage(Collections.singletonList("a language")));
-
-  @GivenTextResource("/domains/verification/v1/webhooks/VerificationResponseSMS.json")
-  String jsonResponseSMS;
-
-  @GivenTextResource("/domains/verification/v1/webhooks/VerificationResponseSMSEmptySMS.json")
-  String jsonResponseSMSEmptySMS;
+  public static PhoneCallRequestEventResponse expectedPhoneCallRequestEventResponseDto =
+      PhoneCallRequestEventResponse.builder()
+          .setAction(VerificationEventResponseAction.DENY)
```

Comment:
Just for the sake of consistency with the API usage: when the action is set to "deny", it doesn't make sense to return some options as the verification will not occur.

Reply:
Done